Adds index files to the new Payment Services Extension folder and subfolders in GraphQL docs (#390)
AdobeDocs/commerce-webapi: src/pages/graphql/payment-services-extension/workflows/index.md

---
title: Payment Services extension workflows
description: Learn how to use the GraphQL APIs to integrate the Payment Services extension.
keywords:
  - GraphQL
  - Payments
---

# Payment Services extension workflows

> The Payment Services extension workflows are detailed guides and documentation on various processes and workflows related to the Payment Services extension. These guides help you understand and implement different features and functionalities provided by the extension, ensuring a seamless integration and optimal usage of the payment services.

Reviewer: ```suggestion
The workflows described in this section provide detailed guides on various processes and workflows related to the Payment Services extension. These guides help you understand and implement different features and functionalities provided by the extension, ensuring seamless integration and optimal usage of Payment Services.
```
